src/components/Dashboard.js

```diff
@@ -18,19 +18,165 @@ import Tooltip from '@material-ui/core/Tooltip';
 import MenuItem from '@material-ui/core/MenuItem';
 import Paper from '@material-ui/core/Paper';
 import NavigationBar from './common/NavigationBar';
+import CardContent from '@material-ui/core/CardContent';
+import Grid from '@mui/material/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+
 
 const pages = ['Events'];
 const settings = ['Update Profile', 'Logout'];
 
+const { REACT_APP_API_ENDPOINT } = process.env;
+
 export default function Dashboard() {
+
+  const { currentUser } = useAuth();
+  const history = useHistory()
+
+  //const email = currentUser.email;
+  const [email, setEmail] = React.useState('');
+  const [userID, setUserID] = React.useState('');
+  const [open, setOpen] = React.useState(false);
+
+  const [events, setEvents] = useState([]);
+
+  React.useEffect(() => {
+    setEmail(currentUser.email);
+    loadUserEmailSearch(currentUser.email);
+  }, []);
+
+  const loadUserEmailSearch = (email) => {
+    callApiGetUserEmailSearch(email)
+      .then(res => {
+        var parsed = JSON.parse(res.express);
+        //console.log(parsed[0].id);
+        setUserID(parsed[0].id);
+      });
+  }
+
+  const callApiGetUserEmailSearch = async (email) => {
+    const url = `${REACT_APP_API_ENDPOINT}/userSearchByEmail`;
+    console.log(url);
+
+    const response = await fetch(url, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        email: email
+      })
+    });
+    const body = await response.json();
+    if (response.status !== 200) throw Error(body.message);
+    return body;
+  }
+
+  const callApiGetRecentEvents = async () => {
+
+    const url = `${REACT_APP_API_ENDPOINT}/getRecentEvents`;
+    console.log(url);
+
+    const response = await fetch(url, {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+      }
+    });
+    const body = await response.json();
+    if (response.status !== 200) throw Error(body.message);
+    return body;
+  }
+
+  const loadGetRecentEvents = async () => {
+    try {
+      const res = await callApiGetRecentEvents();
+      const parsed = JSON.parse(res.express);
+      setEvents(parsed);
+    } catch (error) {
+      console.error(error);
+    }
+  }
+
+  useEffect(() => {
+    loadGetRecentEvents();
+  }, []);
+
+  const useStyles = makeStyles((theme) => ({
+    formControl: {
+      margin: theme.spacing(1),
+      minWidth: 120,
+    },
+    selectEmpty: {
+      marginTop: theme.spacing(2),
+    },
+    root: {
+      '& > *': {
+        margin: theme.spacing(1),
+        width: '50ch',
+      },
+    },
+    paper: {
+      padding: theme.spacing(2),
+      textAlign: 'center',
+    },
+  }));
+
+  const classes = useStyles();
```

Comment:
this isn't being used

Reply:
i think i removed it in the updated branch my rsvp sorry for the confusion